### PR TITLE
Run e2e test for devconsole operator

### PR DESCRIPTION
**Pull request by baijum**

### CODE REVIEW

File: ci-operator/jobs/redhat-developer/devconsole-operator/redhat-developer-devconsole-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -34,6 +34,63 @@ presubmits:
...
+            configMapKeyRef:
+              key: redhat-developer-devconsole-operator-master.yaml
+              name: ci-operator-master-configs
+        - name: JOB_NAME_SAFE
+          value: e2e
+        - name: TEST_COMMAND
+          value: make test-e2e
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
```
Comment (DhritiShikhar) [inline ci-operator/jobs/redhat-developer/devconsole-operator/redhat-developer-devconsole-operator-master-presubmits.yaml]: @baijum why is this `name` field empty?

Comment (baijum) [reply ci-operator/jobs/redhat-developer/devconsole-operator/redhat-developer-devconsole-operator-master-presubmits.yaml]: This file is auto-generated. Not sure why it's empty. I only modified: `ci-operator/config/redhat-developer/devconsole-operator/redhat-developer-devconsole-operator-master.yaml`

Comment (petr-muller) [reply ci-operator/jobs/redhat-developer/devconsole-operator/redhat-developer-devconsole-operator-master-presubmits.yaml]: Prow does not need `name` to be set

**Comment by sbose78** [comment]
@droslean @petr-muller This is ready for merge :)

**Pull request merged**